src/shared/components/icons/svgdefs.svg

```diff
@@ -54,5 +54,17 @@
         <g id="clear" viewBox="0 0 1024 1024">
             <path class="path1" d="M810.667 273.707l-60.373-60.373-238.293 238.293-238.293-238.293-60.373 60.373 238.293 238.293-238.293 238.293 60.373 60.373 238.293-238.293 238.293 238.293 60.373-60.373-238.293-238.293z"></path>
         </g>
+        <g id="chevron-right" viewBox="0 0 1024 1024">
+            <path class="path1" d="M323.525 752.112l253.021-253.021-253.021-253.021 77.455-77.455 330.475 330.475-330.475 330.475z"></path>
+        </g>
+        <g id="check" viewBox="0 0 1024 1024">
+            <path class="path1" d="M384 690l452-452 60 60-512 512-238-238 60-60z"></path>
+        </g>
+        <g id="clear" viewBox="0 0 1024 1024">
+            <path class="path1" d="M810 274l-238 238 238 238-60 60-238-238-238 238-60-60 238-238-238-238 60-60 238 238 238-238z"></path>
+        </g>
+        <g id="add" viewBox="0 0 1024 1024">
+            <path class="path1" d="M899.725 566.645h-333.077v333.077h-109.292v-333.077h-333.077v-109.292h333.077v-333.077h109.292v333.077h333.077v109.292z"></path>
+        </g>
```

Comment:
There's already an icon with the clear id.

They seems identical: http://codepen.io/dinoboff/pen/GJaGmv